docs/docs/getting_started/installation/other_install_methods.md

```diff
@@ -1,38 +1,102 @@
 ---
 title: Alternative Install Methods
-description:
-  There are different ways to install Nargo, the one-stop shop and command-line tool for developing Noir programs. This guide explains other methods that don't rely on noirup, such as compiling from source, installing from binaries, and using WSL for windows
+description: There are different ways to install Nargo, the one-stop shop and command-line tool for developing Noir programs. This guide explains other methods that don't rely on noirup, such as compiling from source, installing from binaries, and using WSL for windows
 keywords: [
-   Installation
-   Nargo
-   Noirup
-   Binaries
-   Compiling from Source
-   WSL for Windows
-   macOS
-   Linux
-   Nix
-   Direnv
-   Shell & editor experience
-   Building and testing
-   Uninstalling Nargo
-   Noir vs code extension
-]
+    Installation
+    Nargo
+    Noirup
+    Binaries
+    Compiling from Source
+    WSL for Windows
+    macOS
+    Linux
+    Nix
+    Direnv
+    Shell & editor experience
+    Building and testing
+    Uninstalling Nargo
+    Noir vs code extension,
+  ]
 sidebar_position: 1
 ---
 
+## Encouraged Installation Method: Noirup
 
-## Installation
+Noirup is the endorsed method for installing Nargo, streamlining the process of fetching binaries or compiling from source. It supports a range of options to cater to your specific needs, from nightly builds and specific versions to compiling from various sources.
 
-The most common method of installing Nargo is through [Noirup](./index.md)
+### Installing Noirup
+
+First, ensure you have `noirup` installed:
+
+```sh
+curl -L https://raw.githubusercontent.com/noir-lang/noirup/main/install | bash
+```
+
+### Fetching Binaries
+
+With `noirup`, you can easily switch between different Nargo versions, including nightly builds:
+
+- **Nightly Version**: Install the latest nightly build.
+
+  ```sh
+  noirup --version nightly
+  ```
+
+- **Specific Version**: Install a specific version of Nargo.
+  ```sh
+  noirup --version <version>
+  ```
+
+### Compiling from Source
```

Comment:
Too late now, as this has been merged, but I would remove this section as it is part of `noirup` documentation (and the CLI)